mremoteng.json

```diff
@@ -11,11 +11,18 @@
             "mRemoteNG"
         ]
     ],
+    "persist": [
+        "confCons.xml",
+        "confCons.xml.backup",
+        "pnlLayout.xml",
+        "extApps.xml",
+        "Themes.xml"
+    ],
     "checkver": {
-        "url": "https://github.com/mRemoteNG/mRemoteNG/releases/latest",
-        "re": "/releases/download/(?<short>[^/]+)/mRemoteNG-Portable-(?<version>(?:\\d+\\.)+\\d+).zip"
+        "github": "https://github.com/mRemoteNG/mRemoteNG",
+        "re": "/releases/download/(?<short>[^/]+)/mRemoteNG-Portable-(?<version>[\\d.]+).zip"
```

Comment:
Does a `.` inside a `[]` automatically get treated as a literal, or are you accidentally matching arbitrary characters? If the former, this won't work, because you have two `.` characters back to back at the end of the version group (`1234..zip`).